### DISCUSSION TITLE
Resources for learning RabbitMQ and pika

### DISCUSSION THREAD

**Discussion by wigging** [pika/pika#1483] [General]
I'm aware of the [RabbitMQ tutorials](https://www.rabbitmq.com/tutorials) and the [pika documentation](https://pika.readthedocs.io/en/stable/) which provide many short examples. But are there any Python books, online tutorials, courses, etc. that focus on using RabbitMQ and pika especially in production environments?

**Comment by wigging** [comment]
I found a [RabbitMQ in Depth](https://www.amazon.com/RabbitMQ-Depth-Gavin-M-Roy/dp/1617291005) book on Amazon but the description doesn't mention pika. The author is @gmr so maybe he can give more details about the book.

**Comment by gmr** [comment]
I don't go into specifics with Pika, but the content should translate well. You should walk away with a deep understanding of AMQP 0.9.1 and how it applies to RabbitMQ and that should make picking up Pika pretty easy.

**Reply by wigging** [reply]
So the "RabbitMQ in Depth" book uses Python and pika for code examples discussed in the book?
